### DISCUSSION TITLE
Template popup

### DISCUSSION THREAD

**Discussion by FutureBuddha** [swellstores/community#555] [Ideas]
![image](https://github.com/user-attachments/assets/f0c2d924-b67a-46a0-b3a0-2042f5fd3b61)

Hi there,

I have noticed an issue with the email body field in the notifications settings. The field is so small that it is almost unusable. This problem occurs when you go to `Settings` -> `Notifications` and try to edit any email template.

I am using the Brave browser, and this issue significantly impacts the usability of the email template editor.

Could you please make the email body field larger to enhance the user experience?

Thank you very much for your assistance.

Best regards

**Comment by sags95** [comment]
The team is aware of this issue and currently working on a fix. We'll be sure to comment here when a fix is released.

**Reply by sags95** [reply]
A fix has been deployed to correct this issue.
